### ISSUE TITLE
Add Post numbers to posts?

### ISSUE THREAD

**Issue by akinsey** [epochtalk/epochtalk#514]
"There is no post number in the personal post history. If there are campaigns, counting posts will be a bit more difficult in relation to how it is done on Bitcointalk."

We can also display the post numbers in posts. I think things get wonky when posts are deleted though... [1, 2, 3, 4, 5] if 4 is deleted post numbers are [1,2,3,5]

**Comment by DdmrDdmrAlt2** [comment]
Bitcointalk renumerates the post numbers if I'm right (preserving unaltered the post Id logically). On my current campaign, The manager uses the post number as a reference to determin the number of posts in  week (i.e. started week on post number 1122 and ended on 1172).

**Comment by akinsey** [comment]
Scratch that the posts do rearrange when a post is purged, just not when hidden. If the post is hidden its still technically in the thread. I added the post position in my last commit, it will still show the post position for hidden threads. I also updated the message so there's no confusion with deleting/hiding. Hidden posts should now display as "Post Hidden By User" with the post position in the right hand corner. 
@DdmrDdmrAlt2
